message_definitions/v1.0/uAvionix.xml

```diff
@@ -4,6 +4,7 @@
   <!-- company URL: http://www.uAvionix.com                       -->
   <!-- email contact: matt@uAvionix.com or jeff@uAvionix.com      -->
   <!-- mavlink ID range: 10000 - 10099                            -->
+  <include>common.xml</include>
```

Comment:
I don't think this is right.  Best check with uavionix, but this change didn't come from them

Reply:
I will remove this commit, and rebase